examples/js/loaders/OBJLoader.js

```diff
@@ -2,6 +2,11 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
+THREE.ParseException = function(message) {
```

Comment:
It's thrown later when an invalid line is encountered. It's so you can catch it and use the `message` property on it. Very convenient.